Add "Mobile screen readers"

jmuheim opened the pull request:
For the records: I've taken many of the following todos from #172.

Todo:

- [x] I think that Mobile is much easier to start with for beginners, so I think it should be placed before Desktop (in Knowledge) **UPDATE:** Yes, I placed it before desktop.
    - [x] As we are now introducing the most basic things about screen readers in Mobile already, there may be redundant information in the subsequent pages about Desktop. It may be sensible to leave some of the redundancy there, but we should add references where possible to Mobile, so the user can relate well between both topics. **UPDATE:** Done.
- [x] Should VO/iOS and Talkback be before or after NVDA and JAWS (in Setup?) **UPDATE:** I chose to put it before, as mobile screen readers are so much easier to get up and running.
- [x] More gestures? -> I think we should only explain gestures that are relevant for web browsing **UPDATE:** Yes, did only explain the very basics.
    - [x] Triple 3 fingers tap (screen dim) **UPDATE:** Done.
    - [ ] ~~There might be more usable gestures, e.g. for interacting with form controls, or to jump to specific elements on a page (Item chooser, triple tap with 2 fingers)... But are they necessary?~~ Could be nice at a later time
- [x] While desktop screen readers (NVDA/JAWS) are very similar to use (so we can write pretty generic texts targeting both of them), VO/iOS and Talkback in some aspects are similar (general use of left/right/double-click), but in others are rather different (quick-usage of on screen keyboard, rotor, more advanced gestures of Talkback, etc.). **UPDATE:** Tried to keep all explanations as generic as possible and as specific as needed. Unsure yet how detailed the explanations for advanced mobile screen reader features should be...?
    - [x] Should we have specific pages for both VO/iOS and Talkback? Or should we pack everything together, but explain when something is specific to one of them? **UPDATE:** Decided to have separate pages in both Setup and Knowledge parts.
- [x] The good news: existing examples have already been tested on mobile (for release 1.0)! 👍 
- Mobile pages need to be filled with Talkback related content.
    - [x] In the setup section: `/setup/screen-readers/talkback/`. **Donato**
    - In the knowledge section:
        - [x] `/knowledge/screen-readers/mobile/reading-websites/` **Donato**
        - [x] `/knowledge/screen-readers/mobile/interacting-with-websites/` **Donato**
- Page "Relevant combinations of screen readers and browsers" needs to be updated
    - [x] There's a newer WebAIM survey meanwhile: https://webaim.org/projects/screenreadersurvey8/ **Esther**
    - [x] We can add VoiceOver/iOS and Talkback specific info
    - [x] We can strip out IE (finally!!! 👍) --> No, don't strip out, better add a side note that IE is still here for documentation purposes and examples are still backwards compatible, but it's not a required combo anymore!
- [x] We can remove IE from the rest of the guide now --> Not yet! Create new ticket **Josh** --> Done: #275
- [x] Fix any broken internal links (and possibly changed titles!) **Jürgen**
- [x] Find any other old references to screen readers in general and make sure it's appropriate to both mobile and desktop! **Nicola**
- [x] Update all "changed" attributes before merge! **Josh** --> Set to `2020-04-07`.
- [ ] ~~Update `netlify.toml` redirects **Josh**~~ Doesn't exist yet. Will have to deal with this at a later time.

jmuheim committed:
Make desktop/mobile agnostic

jmuheim committed:
Optimise content

jmuheim committed:
Merge branch 'master' into content/mobile-screen-readers-section

jmuheim committed:
Create new section "Screen Readers" (mobile/desktop agnostic)

jmuheim committed:
Rename remaining desktop screen reader folder

jmuheim committed:
Update meta data

jmuheim committed:
Edit existing content

jmuheim committed:
Adding content, refactoring content

jmuheim committed:
More content

jmuheim committed:
Add content

jmuheim committed:
Add content

jmuheim committed:
Add content

jmuheim committed:
Change title back

jmuheim committed:
Add content

jmuheim committed:
Add content

jmuheim committed:
Add more content

rudigier committed:
feat: change route for screen reader links

jmuheim committed:
Update "changed" field for updated pages

jmuheim committed:
Merge branch 'content/mobile-screen-readers-section' of https://github.com/Access4all/adg into content/mobile-screen-readers-section

ralf57 commented:
@jmuheim please find the complementary PR regarding TalkBack at https://github.com/Access4all/adg/pull/281

jmuheim commented:
Thank you, I will take a look at it.

> On 23 Apr 2020, at 10:53, Donato Rotunno <notifications@github.com> wrote:
> 
> 
> @jmuheim <https://github.com/jmuheim> please find the complementary PR regarding TalkBack at #281 <https://github.com/Access4all/adg/pull/281>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/Access4all/adg/pull/274#issuecomment-618272269>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAN3DR6TWBK3CQJZGOCD5YDRN76YZANCNFSM4MBALHDA>.
>

ralf57 committed:
TalkBack setup and usage (#281)

* Issue-238: define git branching and deploy strategy

* Issue-238: adapt branching strategy description

* Issue-238: fix typo

* Issue-238: CS fix

* Bump minimist from 1.2.2 to 1.2.3

Bumps [minimist](https://github.com/substack/minimist) from 1.2.2 to 1.2.3.
- [Release notes](https://github.com/substack/minimist/releases)
- [Commits](https://github.com/substack/minimist/compare/1.2.2...1.2.3)

Signed-off-by: dependabot[bot] <support@github.com>

* Change content on 404 page, refs. #247 (#264)

* feat: change content on 404 page, refs. #247

* Add email and remove comma

Co-authored-by: Jürgen Rudigier <jru@hinderlingvolkart.com>
Co-authored-by: Joshua Muheim <josh@josh.ch>

* Whitespace

* More whitespace

* Updated TalkBack setup section

* Updated TalkBack usage sections

* Bump jquery from 3.4.1 to 3.5.0

Bumps [jquery](https://github.com/jquery/jquery) from 3.4.1 to 3.5.0.
- [Release notes](https://github.com/jquery/jquery/releases)
- [Commits](https://github.com/jquery/jquery/compare/3.4.1...3.5.0)

Signed-off-by: dependabot[bot] <support@github.com>

* Revising Donato's TalkBack additions

* Revising

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Jürgen Rudigier <j.rudigier@gmail.com>
Co-authored-by: Jürgen Rudigier <jru@hinderlingvolkart.com>
Co-authored-by: Joshua Muheim <josh@josh.ch>

rudigier committed:
Broken Links (#277)

* fix: broken links

* Revising

Co-authored-by: Jürgen Rudigier <jru@hinderlingvolkart.com>
Co-authored-by: Joshua Muheim <josh@josh.ch>

estherbrunner committed:
Update page on relevant combinations according to latest WebAIM survey (#276)

* Update page on relevant combinations according to latest WebAIM survey

* Revising

Co-authored-by: Joshua Muheim <josh@josh.ch>

Aerzas committed:
Mobile screen readers section scope (#278)

* Emphasize desktop/mobile context when talking about screen readers

* Revising

Co-authored-by: Joshua Muheim <josh@josh.ch>

Aerzas committed:
Update changed dates (#278)

moltebeeri committed:
Update README.md

Corrected typing error (interacht, should be interact)

moltebeeri commented on pages/knowledge/screen-readers/mobile/interacting-with-websites/README.md:
```diff
@@ -0,0 +1,26 @@
+
+To make life easier: while interacting with any standard HTML control, we advise you to stop the screen reader, and after you have completed your interaction, simply restart it. You do not miss any important screen reader interaction here (as you are solely interacting with the system's standard controls).
+
+For example, when entering a text into a textarea, simply move the screen reader cursor to the textarea, then double click to activate it. Navigating the now displayed on-screen keyboard is pretty long-winded using the screen reader, so simply disable the it. Now enter some text as usual, then restart the screen reader and continue with your testing.
```
"simply disable the it." > Something is wrong here. Should probably be "simply disable it" or "simply disable the screen reader". Please verify.

moltebeeri committed:
Update README.md

corrected typing error (dozens OF keyboard shortcuts, not dozens OR keyboard shortcuts)

moltebeeri commented on pages/knowledge/screen-readers/desktop/browse-focus-modes/README.md:
```diff
@@ -89,7 +89,7 @@ To make listening to the `Tab` key work again, a corresponding functionality nee
 
 Also, while leaving focus mode is always possible using the `Esc` key, this is not available in application mode as well. In the worst case, screen readers get trapped within such an element and will not be able to get out of it anymore. So again, this needs to be implemented using JavaScript - together with each and every other needed interactivity.
```
Sounds a bit strange, the "not available ... as well" part. Should it not be something like "Also, while leaving focus mode is always possible using the `Esc` key, this option is not available in application mode." Or maybe even without the "Also", so "While leaving focus mode is always possible using the `Esc` key, this option is not available in application mode."

jmuheim replied:
Fixed. 👍

jmuheim replied:
The "as well" part should emphasise that this is one of the many things that are not available in application mode. I fixed it as you proposed, and added "(like many other options)".

moltebeeri replied:
All fine now - and it contains what you wanted to emphasise!

moltebeeri reviewed (commented):
Found 2 parts where I'd suggest corrections.

jmuheim committed:
Typo, thanks @ManuA4A!

moltebeeri committed:
Update README.md

Word order was wrong, "which can usually be configured" instead of "which can be usually configured under"

jmuheim committed:
Updated as proposed by @ManuA4A.

jmuheim committed:
Merge branch 'master' into content/mobile-screen-readers-section

moltebeeri committed:
Update README.md

There ARE lots of add-ons instead of There is ...

jmuheim committed:
Add info about speech reader and caption panel (#284)

moltebeeri reviewed (approved):
All read, some minor corrections done.
